CONTRIBUTING.rst

```diff
@@ -211,7 +211,7 @@ but here is a simple overview:
    You need to have Python 2.7 and 3.5 available in your system.  Now
    running tests is as simple as issuing this command::
 
-    $ python runtox.py -e linting,py27,py35
+    $ python3 runtox.py -e linting,py27,py35
```

Comment:
This change is already on `master`, cherry-picked here to see green builds. :grin: